sections/createaservice/createaservice.md

```diff
@@ -5,24 +5,35 @@ description: Introduction to how to create a service in Veracity.
 
-All services in Veracity needs to be part of a project in the Provider Hub, or My Projects, located on the menu bar in Developer.veracity.com. From there, resources like Veracity Identity can be provisioned self-service.
+All services in Veracity needs to be part of a project in the Provider Hub. You can find your projects located on the menu bar in developer.veracity.com. Resources like application credentials used for Veracity Identity can be provisioned self-service from here.
 
-A project in Veracity Provider Hub is what holds all resources related to your product. You can add collaborators into a Project, and assign access rights to these collaborators. You would typically have one project for every product. E.g., the DNV GL product <i>DNVGL Oil and Gas Standards and Recommended Practices</i>, would usually have one project called <i>DNVGL Oil and Gas Standards and Recommended Practices</i>.
+A project in Veracity Provider Hub is what holds all resources related to your product. Resources are organized in resource groups. You can add collaborators into a Project, and assign access rights to these collaborators. You would typically have one project for every product. E.g., the DNV GL product DNVGL Oil and Gas Standards and Recommended Practices, would usually have one project called DNVGL Oil and Gas Standards and Recommended Practices.
 
 Inside a project, you can create a resource. A resource can be application credentials for using Veracity Identity in your service or any other platform capabilities. You can think of a project as a resource group, with some user management.
 
 
-## What is a resource
-A resource is any platform capability that needs provisioning for your project to use it. An example of such a resource is Application credentials, which is required in order to integrate with the Veracity Identity.
+## What are resources and resource groups?
 
-Some of the resources you can provision self-service, some are default part of a project, and some are configured by Veracity Onboarding before your product is released to production on Veracity. e.g. Marketplace is provisioned by Veracity Onboarding and will be done in close collaboration with you.
+A resource is any platform capability that needs provisioning for your project to use it. An example of a resource is Application credentials, which is used to register your application toward the Veracity Identity Provider so that you can log in with Veracity.
+
+Some of the resources you can provision self-service,  part of a project by default, and some are configured by Veracity Onboarding before your product is released to production on Veracity. e.g. Marketplace is provisioned by Veracity Onboarding and will be done in close collaboration with you.
+
+A resource group is used to collect a set of resources. Currently the resource group will be labeled with environments, such as “devtest”, “test”, “staging or “production”. 
+
+
+<figure>
+	<img src="/ProjectStructure.png"/>
```

Comment:
You need to create a folder called `assets` in the folder `createaservice`. In this folder you put this image. Then you point to it like this: `./assets/ProjectStructure.png`